templates/lxd/index.html

```diff
@@ -0,0 +1,299 @@
+        </div>
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5">Secure and scalable</h3>
+          <p>LXD runs unprivileged containers by default &mdash; protecting the host system from potential attacks. For virtual machines, LXD uses modern virtual hardware (VirtIO) exclusively. In addition, it utilises UEFI SecureBoot and provides vTPM support.</p>
+          <p>Resource restrictions through cgroups and ulimits, as well as RBAC, are also supported.</p>
+          <p>LXD is also easy to scale &mdash; from containers on your laptop to thousands of instances in the data centre.</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="u-fixed-width p-section--shallow">
+		<hr class="p-rule">
+		<h2>Alternatives for running workloads</h2>
+	</div>
+  <div class="row--25-75">
+    <div class="col">
+      <hr class="p-rule u-hide--medium u-hide--small">
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5" style="border-top: 3px solid #000;">System containers</h3>
+        </div>
+        <div class="col-6 col-medium-4">
+          <ul class="p-list--divided u-no-padding--bottom">
+            <li class="p-list__item is-ticked">To run a full Linux OS inside a container</li>
+            <li class="p-list__item is-ticked">Utilises Kernel of the host</li>
+            <li class="p-list__item is-ticked">Identical performance to bare metal</li>
+          </ul>
+        </div>
+      </div>
+      <hr class="p-rule">
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5" style="border-top: 3px solid #000;">Virtual machines</h3>
+        </div>
+        <div class="col-6 col-medium-4">
+          <ul class="p-list--divided u-no-padding--bottom">
+            <li class="p-list__item is-ticked">For workloads needing a different kernel or OS than the host</li>
+            <li class="p-list__item is-ticked">Legacy free</li>
+            <li class="p-list__item is-ticked">Cloud-like experience</li>
+          </ul>
+        </div>
+      </div>
+      <hr class="p-rule">
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5" style="border-top: 3px solid #000;">Clusters for more demanding workloads</h3>
+        </div>
+        <div class="col-6 col-medium-4">
+          <ul class="p-list--divided u-no-margin--bottom">
+            <li class="p-list__item is-ticked">Up to 50 servers in a unified cluster with 1000s of instances</li>
+            <li class="p-list__item is-ticked">Run containers, VMs, or a combination of the two</li>
+            <li class="p-list__item is-ticked">Same distributed database</li>
+            <li class="p-list__item is-ticked">Managed uniformly</li>
+            <p>Watch this <a href="https://www.youtube.com/watch?v=nrOR6yaO_MY">deep dive into LXD clustering</a></p>
```

Comment:
```suggestion
            <p><a href="https://www.youtube.com/watch?v=nrOR6yaO_MY">Watch this deep dive into LXD clustering</a></p>
```
On the doc the whole of the text is hyperlinked

Reply:
fixed